css/wee.module.less

```diff
@@ -16,9 +16,12 @@
 .wee-{{moduleName}}-responsive () when (@responsiveEnabled = true) {
 	.check () when not (@mobileLandscapeWidth = false) {
 		@media (min-width: (@mobileLandscapeWidth)) {
-			html {
-				font-family: '"2"';
+			.check () when ({{standalone}} = true) {
```

Comment:
In most cases that would be correct, yes. The solution didn't excite me but the problem of the breakpoint output for every module had to be solved one way or another.